app/src/main/java/net/gsantner/markor/util/DocumentIO.java

```diff
@@ -11,31 +11,42 @@
 
 import android.content.Context;
 import android.content.Intent;
+import android.os.Build;
 import android.os.Bundle;
 import android.support.annotation.Nullable;
+import android.support.annotation.RequiresApi;
 import android.text.InputFilter;
 import android.text.Spanned;
 import android.text.TextUtils;
+import android.util.Log;
+import android.widget.Toast;
 
+import net.gsantner.Constants;
 import net.gsantner.markor.R;
 import net.gsantner.markor.activity.MainActivity;
 import net.gsantner.markor.format.TextFormat;
 import net.gsantner.markor.format.markdown.MarkdownTextConverter;
 import net.gsantner.markor.model.Document;
+import other.de.stanetz.jpencconverter.JavaPasswordbasedCryption;
+import other.de.stanetz.jpencconverter.PasswordStore;
 import net.gsantner.opoc.util.FileUtils;
 
 import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.nio.charset.StandardCharsets;
+import java.security.SecureRandom;
 import java.util.Locale;
 import java.util.UUID;
 
 public class DocumentIO {
     public static final String EXTRA_DOCUMENT = "EXTRA_DOCUMENT"; // Document
     public static final String EXTRA_PATH = "EXTRA_PATH"; // java.io.File
     public static final String EXTRA_PATH_IS_FOLDER = "EXTRA_PATH_IS_FOLDER"; // boolean
+    private static final String LOG_TAG_NAME = "DocumentIO";
```

Comment:
Please use `Log.d(getClass().getName(), "TEXT");` instead of introducing variable